### ISSUE TITLE
Support for region us-gov-west-1

### ISSUE THREAD

**Issue by wbwatkinson** [determined-ai/determined#1860]
Attempting to run det-deploy in region `us-gov-west-1` yields the following error:
```
det-deploy is only supported in ['ap-northeast-1', 'eu-central-1', 'eu-west-1', 'us-east-1', 'us-east-2', 'us-west-2'] - tried to deploy to us-gov-west-1
use the --region argument to deploy to a supported region
```

Recommended fix: allow us-gov-west-1 as a region for det-deploy

**Comment by vishnu2kmohan** [comment]
Thanks for the suggestion @wbwatkinson, we'll keep you posted.

**Comment by wbwatkinson** [comment]
Perfect! Thank you. Just so you are aware, I'll be dead in the water for using the software until I can deploy to us-gov-west-1, because we can put cloud services anywhere other than in the gov regions. I suspect many DoD and government agencies will have a similar restriction.

**Comment by vishnu2kmohan** [comment]
Got it! We're actively working to get on-boarded onto GovCloud and hope to have some updates soon.

**Comment by vishnu2kmohan** [comment]
@wbwatkinson FYI - We _just_ [added support](https://github.com/determined-ai/determined/commit/fc7fee623519770901a21f14d101b29eefcd2f35) for GovCloud to `det-deploy`, so it should be available in the next release: 0.14.4 (in about a couple of weeks, and not in the one that will be going out soon - i.e., 0.14.3).

**Comment by vishnu2kmohan** [comment]
Closing out, given we landed official support for GovCloud in 0.14.5 and a related [fix](https://github.com/determined-ai/determined/commit/92056b569e19e5ed0d865dfd94b3cc5e7136bf12) will land in the upcoming release (0.15.0).
